Sources/XCHammer/XcodeTarget.swift

```diff
@@ -809,8 +843,13 @@ public class XcodeTarget: Hashable, Equatable {
                 OrderedArray(["$(SRCROOT)/external/"])
 
         // Add copts for module maps
-        settings.copts <>= self.ruleEntry.objCModuleMaps.map {
-            "-fmodule-map-file=" + subBazelMakeVariables(getRelativePath(for: $0,
+        settings.copts <>= self.ruleEntry.objCModuleMaps.compactMap { map in
+            // FIXME(rules_ios) - this is a bug, we pick these up and it causes an issue
+            if map.subPath.hasSuffix(".swift.modulemap") {
```

Comment:
Ok, so filtering this out should be fine? What I was trying to do was to make them get into `xchammer-includes` so Xcode could find them but I guess this is not necessary?